src/whylogs/core/image_profiling.py

```diff
@@ -0,0 +1,124 @@
+from PIL import Image
+
+from PIL.ExifTags import TAGS as eTAGs
+
+from PIL.TiffTags import TAGS
+from PIL.TiffImagePlugin import IFDRational
+from whylogs.core.datasetprofile import DatasetProfile
+
+from typing import Optional, Callable, List, Union, Dict
+import numpy as np
+
+from whylogs.features import _IMAGE_FEATURES
+from whylogs.features.transforms import Hue, Saturation, Brightness
+
+
+DEFAULT_IMAGE_FEATURES = [Hue(), Saturation(), Brightness()]
+
+
+_METADATA_DEFAULT_ATTRIBUTES = [
+    "ImageWidth",
+    "ImageLength",
+    "BitsPerSample",
+    "Compression",
+    "Quality",
+    "PhotometricInterpretation"
+    "SamplesPerPixel",
+    "Model",
+    "Software",
+    "ResolutionUnit",
+    "X-Resolution",
+    "Y-Resolution",
+    "Orientation",
+    "RowsPerStrip",
+    "ExposureTime",
+    "BrightnessValue",
+    "Flash",
+]
+
+
+def image_loader(path: str = None)-> Image:
+
+    with open(path, "rb") as file_p:
+        img = Image.open(file_p).copy()
+        return img
+
+
+# def init():
+
+#     for feature in _IMAGE_FEATURES:
+#         __import__(f"")
+
+class TrackImage:
+
+    def __init__(self,
```

Comment:
can you add some documentation here?

Reply:
done in e558a68d9c10385fa8853b8215ad94a03337557b